Modules/SubModule_Population/xASL_adm_GetPopulationSessions.m

```diff
@@ -51,8 +51,9 @@
 else % If files found, continue with defining sessions from SessionList
     
     for iSession = 1:numel(SessionList)
+        [~, folderName] = fileparts(SessionList{iSession});
         [IndexStart, IndexEnd] = regexpi(SessionList{iSession}, 'ASL_\d+');
-        NewList{iSession,1} = SessionList{iSession}(1,IndexStart:IndexEnd); % create a cell array containings characters of all session numbers present
+        NewList{iSession,1} = SessionList{iSession}(1,IndexStart(end):IndexEnd(end)); % create a cell array containings characters of all session numbers present
```

Comment:
It is good to go for the last occurrence. Cause in theory, there could be more occurrences. However, I don't understand how this solves what is written in the issue. Shouldn't you rather modify 'ASL_\d+' to '^ASL_\d+$'?

Reply:
See my final attempt to explain below :)